source/README.md

```diff
@@ -4,12 +4,26 @@ Redis Event Source for Knative
 
 ## Getting started
 
+Prior to installing the source, you will need to decide whether you need to add your TLS Certificate that lets you securely connect to your Redis instance (for example, an instance of Redis on IBM Cloud) to the [`config-tls`](config/config-tls.yaml) Config Map. You do not need to do this if you are using a local instance of Redis, as described in the Example section below.
+
+Edit the [`config-tls`](config/config-tls.yaml) Config Map to add TLS Cert to the `cert.pem` data key (if necessary):
+
+```
+vi source/config/config-tls.yaml
+```
+
 Install the source:
 
 ```sh
 ko apply -f source/config
 ```
 
+Note: In addition to configuring your TLS Certificate, if you are using a remote instance of Redis, you will need to set the appropriate address in t[`redisstreamsource`](../samples/source/redisstreamsource.yaml) source yaml. Here's an example connection string:
```

Comment:
typo with the extra t?

Reply:
Done (commit bc8d7bacabd96f4817c9eef87ade03ecac171df1)